zanata-war/src/main/webapp/WEB-INF/layout/version/upload_translation_modal.xhtml

```diff
@@ -4,63 +4,87 @@
   xmlns:zanata="http://java.sun.com/jsf/composite/zanata"
   xmlns:a4j="http://richfaces.org/a4j"
   xmlns:rich="http://richfaces.org/rich"
+  xmlns:f="http://java.sun.com/jsf/core"
   class="modal" id="uploadTranslationDialog" tabindex="-1" role="dialog"
   aria-labelledby="myModalLabel" aria-hidden="true">
 
   <script>
     $(document).ready(function() {
```

Comment:
- [ ] are we using `$` or `jQuery`? This file has both, we should be consistent.